src/modules/CRUD/components/DataTable/Header/Header.js

```diff
@@ -51,9 +52,9 @@ const Header = ({
       {showLoader && <Spinner size="20" />}
       {showCount && (
         <Tag intent={Intent.PRIMARY} round>
-          {featuresList.length > 1
-            ? t('CRUD.table.results_plural', { count: featuresList.length })
-            : t('CRUD.table.results', { count: featuresList.length })
+          {count > 1
+            ? t('CRUD.table.results_plural', { count })
+            : t('CRUD.table.results', { count })
```

Comment:
You do not have to do such a test. i18next do it itself: https://www.i18next.com/translation-function/plurals#plurals